backbone/src/main/java/org/researchstack/backbone/ui/step/layout/InstructionStepLayout.java

```diff
@@ -127,14 +124,6 @@ public void refreshStep() {
             String title = step.getTitle();
             String text  = step.getText();
 
-            if (TextUtils.isEmpty(title) &&
-                    !TextUtils.isEmpty(text) && !TextUtils.isEmpty(instructionStep.getMoreDetailText()))
-            {
-                // With no Title, we can assume text and detail text is equla to title and text
```

Comment:
This is the main point of the change. Took a bit to figure out why things were acting broken when I didn't supply a title, even though I didn't want a title.